spec/bowling_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'bowling'
+
+describe Bowling do
+  let(:game) { described_class.new }
+
+  describe '#total_frames' do
```

Comment:
As you are using a very specific format for your bowling game, perhaps add additional tests and appropriate code to ensure the input is in the right format.  Your code will fail if the inner arrays are not in the right format